sdks/java/io/cassandra/src/main/java/org/apache/beam/sdk/io/cassandra/CassandraIO.java

```diff
@@ -499,4 +506,191 @@ public void teardown() throws Exception {
       writer = null;
     }
   }
+
+  /**
+   * A {@link PTransform} to delete into Apache Cassandra. See {@link CassandraIO} for details on
+   * usage and configuration.
+   */
+  @AutoValue
+  public abstract static class Delete<T> extends PTransform<PCollection<T>, PDone> {
```

Comment:
Do you think we can reuse (maybe via inheriting from Write or making a new class to config both) the configuration from `Write`). I am a bit uneasy with the duplicate code.

Reply:
Sure we can probably reuse. This was a first version to have something working quickly and start a discussion.
Thanks for the feedback. I'll improve the patch soon.